packages/richtext-lexical/src/field/features/indent/feature.client.tsx

```diff
@@ -39,19 +39,17 @@ const IndentFeatureClient: FeatureProviderProviderClient<undefined> = (props) =>
               },
               key: 'indent-decrease',
```

Comment:
key naming convention here is different as well, we should keep these consistent